lib/MusicBrainz/Server/Controller/CDTOC.pm

```diff
@@ -57,9 +57,17 @@ sub show : Chained('load') PathPart('')
     my ($self, $c) = @_;
 
     my $cdtoc = $c->stash->{cdtoc};
+    my $medium_cdtocs = $self->_load_releases($c, $cdtoc);
+
+    for my $mc (@$medium_cdtocs) {
+        $mc->perfect_match = $c->model('Medium')->perfect_match_cdtoc(
```

Comment:
Huh, does that work? I'd expect not since `$mc->perfect_match` is a method call...but I know you can set the new value by passing it to `$mc->perfect_match()`.